private-templates-service/server/src/index.d.ts

```diff
@@ -0,0 +1,83 @@
+// Type definitions for adaptivecards-templating-service 1.0
+// Project: https://github.com/microsoft/AdaptiveCards-templates
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+declare module 'adaptivecards-templating-service' {
+    import { Router } from 'express';
+
+    export interface IUser {
```

Comment:
Most of these are duplicates? https://github.com/microsoft/adaptivecards-templates/blob/dev/private-templates-service/types/adaptivecards-templating-service/index.d.ts

Reply:
I added this file when I was debugging the adaptivecards-templating-service module. Just deleted it.